saml2/saml2-service-provider/src/main/java/org/springframework/security/saml2/provider/service/authentication/Saml2AuthenticationRequestContext.java

```diff
@@ -87,6 +98,32 @@ public String getRelayState() {
 		return this.relayState;
 	}
 
+	/**
+	 * Returns the desired ({@code RequestedAuthnContext} that this SP wishes to set in
+	 * the request.This property populates the {@code AuthNRequest.RequestedAuthnContext}
+	 * XML attribute.
+	 * @return the AuthnContextClassRef value
+	 */
+	public RequestedAuthnContext getRequestAuthnContext() {
+		return this.requestAuthnContext;
+	}
+
+	/**
+	 * Returns the isPassive value, if present in the parameters
+	 * @return the isPassive value, default is set to false
+	 */
+	public boolean getIsPassive() {
+		return this.isPassive;
+	}
+
+	/**
+	 * Returns the ForceAuthn value, if present in the parameters
+	 * @return this forceAuthn value, default is set to false
+	 */
+	public boolean getForceAuth() {
```

Comment:
I think this should be called `getForceAuthn` since that's the name of the SAML attribute.